core/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -43,3 +43,9 @@ private[spark] case class SparkUserAppException(exitCode: Int)
  */
 private[spark] case class ExecutorDeadException(message: String)
   extends SparkException(message)
+
+/**
+ * Exception thrown when several InsertDataSource operations are conflicted.
+ */
+private[spark] case class InsertDataSourceConflictException(message: String)
```

Comment:
nit: let's be more specific. It only applies to file source.

Reply:
fix